cinema/urls.py

```diff
@@ -1,10 +1,45 @@
-from django.urls import path
+from django.urls import path, include
 
-from cinema.views import movie_list, movie_detail
+from cinema.views import (
+    MovieViewSet,
+    ActorList,
+    CinemaHallViewSet,
+    GenreList,
+    GenreDetail,
+    ActorDetail,
+)
+from rest_framework import routers
+
+app_name = "cinema"
+router = routers.DefaultRouter()
+router.register("movies", MovieViewSet, basename="movies")
```

Comment:
The `basename` in `router.register` should typically be singular, like `movie`, to match the convention used in Django REST Framework. This helps maintain consistency across the project.